lib/pelion/Kconfig

```diff
@@ -74,6 +74,17 @@ config PELION_NRF_SECURITY
 	help
 	  Default mbedtls configuration for Pelion.
 
+config PELION_QUICK_SESSION_RESUME
```

Comment:
that cannot be part of choice ;)

Reply:
Okay. I wanted to make it appear in menuconfig when the option is enabled, seemed strange but looked nice :D I will change that.

Reply:
done.

Reply:
> wanted to make it appear in menuconfig when the option is enabled

I think you can do that by doing 
```
if PELION_NRF_SECURITY
config ...
  ...
```

Reply:
I left with `depends on`.